pages/docs/visualizing-data/chart-types.mdx

```diff
@@ -47,6 +47,16 @@ Horizontal bar charts are extremely effective data visualizations for comparing
 
 Hashboard tables provide powerful options for visualization, formatting, and grouping. Tables are often the best option when comparing multiple [Measures](/docs/data-modeling/measures) and breakouts at the same time.
 
+## Pie Charts
+
+<img
+  src="/images/pie_chart.png"
+  alt="Three pie charts showing calls to department by borough"
+  style={{ padding: "20px" }}
+/>
+Pie/donut charts depict proportional relationships, and are suited for illustrating
+percentages and distributions of categorical data.
```

Comment:
[nonblocking]

@trucklos any suggestions for copy to guide people more toward a 100% stacked bar chart?